packages/manager/apps/web/client/app/hosting/advices/hosting-pro/advice-hosting-pro.controller.js

```diff
@@ -0,0 +1,148 @@
+import compact from 'lodash/compact';
+import endsWith from 'lodash/endsWith';
+import filter from 'lodash/filter';
+import get from 'lodash/get';
+
+export default class {
+  /* @ngInject */
+  constructor(
+    $http,
+    $q,
+    $state,
+    $translate,
+    Hosting,
+    OvhApiMe,
+    RedirectionService,
+    CORE_URLS,
+  ) {
+    this.$http = $http;
+    this.$q = $q;
+    this.$state = $state;
+    this.$translate = $translate;
+    this.Hosting = Hosting;
+    this.OvhApiMe = OvhApiMe;
+    this.RedirectionService = RedirectionService;
+
+    this.CORE_URLS = CORE_URLS;
+  }
+
+  $onInit() {
+    this.advices = this.loadAdvices();
+  }
+
+  loadAdvices() {
+    return this.OvhApiMe.v6()
```

Comment:
Could we use  `Environment.getUser()` to avoid an API call ?

Reply:
This is done. Thanks :)